buildpipeline/Core-Setup-Linux.json

```diff
@@ -175,12 +181,12 @@
         "SymbolStore"
       ],
       "daysToKeep": 2,
-      "minimumToKeep": 10,
+      "minimumToKeep": 1,
```

Comment:
All of the other definitions except for Fedora23 were specifying 1 so I felt this was what I should use.  I have no opinion - you tell me.